src/code/stores/graph-store.ts

```diff
@@ -53,6 +53,10 @@ interface LogOptions {
   logEvent?: boolean;
 }
 
+interface UndoRedoOptions {
+  skipUndoRedo?: boolean;
+}
+
```

Comment:
```suggestion
type RecordOptions = LogOptions & UndoRedoOptions;
```
Just to DRY up the code below.

Reply:
I kind of like it having the two types in the method signatures - it feels a little more self-documenting to me.  I'm going to leave it in for now.